### ISSUE TITLE
Clean-Up Crew

### ISSUE THREAD

**Issue by Charanraja123** [Gr770/CK3-Community-Title-Project#57]
I'm going to be taking culture by and culture and be cleaning up stuff. I'll list out the culture when I'm starting to clean it up. I do not expect you guys to help, I can kind of do it on my own for the coding side.
At the same, I would need someone to verify if the localizations are correct and make sense. Hopefully @Viridianus can help me out with that part, since he seems to be the most bothered about these stuff along with me xP
This is also going to be a slow process in a way, as I'm going to be doing it at a little leisurely pace. Maybe a single culture for a day kind of thing.

I'll list out the culture as comments as we proceed through them

**Comment by Viridianus** [comment]
Reminder that we will have to change localization files if we change names in 00_CTP - and those will have to be updated across four languages.

**Comment by Charanraja123** [comment]
We can just clean up, add stuff in English only, Maybe before merging Fixes @Gr770 can just copy paste everything (except first line). So that it's easier for all of us. What do you guys think?

**Comment by Viridianus** [comment]
Let's try and trust @Gr770 to do that afterwards :D Hope that issue with RT was an outlier :D OK, kidding aside, that's a possibility, though it's not that difficult for us to do this, either.
Though note that for files with more difficult structure (where some non-titular localizations are in) copy-paste would not work, but for now we seem to work with an easy case.

**Comment by Charanraja123** [comment]
I'm gonna start work on Goidelic group now (I just did a change couple of days ago, but things changed again). 
Give me the particulars for principality/petty kingdom to be used here.

**Comment by Viridianus** [comment]
Irish does not need principalities as its titles are derived from its word for king anyway, Gaelic should use Duke's title (I'm basing myself on this on the Wikipedia: "A mormaerdom was not simply a regional lordship, it was a regional lordship with official comital rank").

**Comment by Viridianus** [comment]
Oh, and also: the "Túatha" we have in the sheet for Irish seems to be plural, while singular lacks the final -a.

**Comment by Viridianus** [comment]
By the way, it seems to be a recurring pattern that Petty King is the same as Duke.

**Comment by Charanraja123** [comment]
So as of now
1) No flavourisations for Irish independent counts/dukes. or should I use the word for King there.
2) Same as independent duke you mean?  Isn't a petty king a independent kingdom? I'm very confused now.

**Comment by Viridianus** [comment]
1. No flavourizations for Irish.
2. Formally, by CKII logic, Petty King _is_ independent duke (hence "Petty" - it's something like a fancy "Small"). In our sheet now, the corresponding row is also used for independent _counts_ (hence why I suggested splitting them).
3. My previous message was about the fact that the said row is often filled with the same words as for Duke (cf., for instance, Welsh: both use Tywysog).

**Comment by Charanraja123** [comment]
Okay. @Gr770  Can we please split them up and name them Principality for one and Petty king for other.
Now does That mean principality doesn't apply to dukes? Or do we need three rows
1.Principality - County
2.Principality - Duchy
3.Petty Kindgom
We even have a Grand duchy to add to all of this lol.

**Comment by Viridianus** [comment]
No, this means that your 2 and your 3 are the same (at least as far as coding and sheet is concerned).

Unrelated to the rows but related to the clean-up: please see my comments on tribal titles for Gaelic.

**Comment by Charanraja123** [comment]
I see you tribe alternatives on the sheet. Should i enter them in? or wait for the other person to comment and then resolve the conversation?

**Comment by Viridianus** [comment]
I don't... actually... know. If we could contact the contributor called Aemr it would be great. For now I'd comment the tribal strings out.

**Comment by Gr770** [comment]
I can split county/duchy when I get home.

**Comment by Charanraja123** [comment]
I've Updated the independent counts. I've added "To be checked" beside all gaelic tribe flavorisations. I know you asked me to comment them out, but commenting them out also required adding tribal government into feudal clan so it doesn;t go to vanilla pdx flavorisaiton which does not make sense as they already use the same thing as Feudal.
Instead I just, added `to be checked` beside each gaelic county tribal localisation for remembrance purposes. 
Other than that everything is good to go i guess.
Move on to the next culture?

**Comment by Viridianus** [comment]
I've checked your comments; I actually wait for Aemr's answers on some more but we can return to that when we reach Brythonic. West Germanic, charge!

**Comment by Charanraja123** [comment]
So first things first. 
From what I see on the sheets
1) No independent county (and derivatives)
2) No petty kingdom for Anglo-Saxon, I think that is something to be added?
3) Also can u check the mayor and city terms for anglo-saxon, they feel weird to me.

In the meantime, I'll check the code and see what surprises await me there

**Comment by Charanraja123** [comment]
HOLY MOTHER OF GOD!!.
This is a mess. 
So much for the Charge, We are in a Swamp lol.
This is gonna be a struggle.

@Viridianus I saw your comments, and this was done before I started working on this project, I was not involved in this.

**Comment by Viridianus** [comment]
1. Well, we don't have the terms but we don't know whether they lack because they lack or they lack because we only added the row, well, this night. But I tentatively think independent earls are a thing.
2. Probably.
3. I have no idea; better ask @Urkaun or Übergold (as Germanic experts) or @palfourti or Alenarae118 - or whoever wrote this. But yeah, I feel like Scir must be somewhere in Anglo-Saxon, too.
As for comments - glad to hear that)

**Comment by Viridianus** [comment]
This discussion of [Brytenwalda](https://docs.google.com/spreadsheets/d/1h_FkusqaFE_uAw0n1HUPFTqg4Ojq8DZqt-sR0eFagyo/edit?disco=AAAAG2Wf-l4) may be very relevant to Anglo-Saxons.

**Comment by Charanraja123** [comment]
Update:
1) I've rearranged Goidelic group to be in line with the sheet. I don't think you have to check goidelic again, since I only re-organised stuff
2) Cleaned up English and Anglo-Saxon. I've added everything except the sultan terms. Also, note that I have yet to touch localization yet, this is all in the main sheet. Government types, cultures and priority numbers should be good for those cultures.
Whenever you have time, Take a peek.
3) I'll try to finish up West Germanic within the next 24 hours

**Comment by Charanraja123** [comment]
Also, I think after West Germanic, we should skip over to Frankish. Since I remember deleting lots of occitan, norman and french cultures

**Comment by Viridianus** [comment]
1. Oh, then you'll have to rearrange each group like that :D
2. Oh, holy guacamole... Yeah, we've probably just deleted some Norman entries. Why are you so fixed against sultan terms, though?
3. I hope.
4. It fully depends on whether we'll get word from Aemr on the questions I sent him in reasonable time. If yes, we should tackle Brythonic first, since it will be directly linked to Goidelic. If no, yes, cleanup in Frankish sounds lovely (though note that this will multiply: they frequently reference other Romance).

Aemr came to the rescue, I see.

**Comment by Viridianus** [comment]
List of titles freed from English dominance over Frankish (for future reference):
empire_english (Norman)
title_prince_male_english (Norman, Occitan- both _kingdom and _empire)
title_prince_female_english (French, Norman - both _kingdom and _empire)
duke_male_english (Norman, Occitan)
duke_female_english (Norman)
baron_republic_male_english (Norman)
baron_republic_female_english (Norman)
count_male_english_independent (Norman, Occitan)
count_female_english_independent (French, Norman)

**Comment by Charanraja123** [comment]
Thanks for the list
Also I'll add sultan in xP. Something just does not feel right about it lol. I'm just kidding
Yep Aemr did come. I was there, blinking at my screen, xD

**Comment by Charanraja123** [comment]
Question: 
Sultan as an King with Islamic religion is acceptable right?

**Comment by Charanraja123** [comment]
Because if you want, I can create custom loc, where it check if he is an Islamic ruler with head of faith, then it gives Sultan or otherwise the regular localisation.

We can add level of piety conditions if we want.
Your call

**Comment by Viridianus** [comment]
@Gr770 - I believe it's a question for you what we should put inside a Sultan.
@Charanraja123 - Am I right that we won't have Sultanates this way for now, only Sultans?

**Comment by Charanraja123** [comment]
Uhh.. We haven't figured out how to do titles. Yep. You are right, I cant do custom loc for this now. :\

**Comment by Viridianus** [comment]
As I thought. Oh, and also: until Pdx manages to do the "shared head of faith" schtick they promised pre-release, Ashari is the only Sunni line which will have a head of faith, so, e.g., Muwalladi Sultans will not be Sultans. Same situation with Shi'a, I gather.

**Comment by Charanraja123** [comment]
Anyway my question still stands in any case. Should I mod it in and how?

Also, Finished West Germanic, Check and let me know when you free
This time I even arranged localisation to the order in the sheets xP

**Comment by Viridianus** [comment]
Again, ask @Gr770 whether he's fine with current Sultans.

OK, code-wise I don't see problems in localization (except for the fact that you threw English Sultans out)))). Title-wise... I think they're all supposed to have Petty Kings not Principalities, but then again, Princehood may represent Norman rules accepted by Englishmen? (Furistos of Old Saxons share a similar problem, but here @Urkaun or Uebergold may help - for English I'm at a loss.)

**Comment by Viridianus** [comment]
Also, [this](https://forum.paradoxplaza.com/forum/threads/a-few-things-about-the-anglo-saxons.1418985/) seems to be relevant, even if I don't fully agree.

**Comment by Gr770** [comment]
Principalities in england came from Wales. The Prince of Wales is a prince because he rules over Wales not because he is royalty. Tywysog means prince/king in welsh.

**Comment by Viridianus** [comment]
Well, yeah, there's that (pretty obvious once you stop and think, and I kinda feel stupid now), but the question is: should English culture actually have principalities or petty kingdoms?

**Comment by Gr770** [comment]
Correction: it means leader but the word went Tywysog > Princeps > Prince thus you can make the argument that 'latin' Normans influenced it to be Principalities rather than Kingdoms

**Comment by Gr770** [comment]
Sultans are muslim independent rulers. Both powerful and sovereign without claiming to be Caliph. Its kinda like princes and principalities.

I think just making king independent plus Islam is fine. Maybe down the line we can work it into a question for culture group of the week.

**Comment by Viridianus** [comment]
Aha, so we'll need to change every of our king_..._muslim to king_..._independent_muslim and add "only_independent = yes"?

**Comment by Gr770** [comment]
As far as I know Sultan as governors under the caliph was an ottoman only thing so yeah.

**Comment by Viridianus** [comment]
Especially given that Caliphates are technically duchies now...

**Comment by Charanraja123** [comment]
From What I read, Sultans were kings who owed some amount of subservience to the Caliph (Especially in religious decisions). Obviously we can't do that now, because Mr.Paradox wont allow flags for some reason. 
So either we can wait for paradox to Introduce those, or we can just say king_independent_muslim, which might be slightly inaccurate, but at the end of the day adds more flavourisation.

**Comment by Charanraja123** [comment]
>  I think they're all supposed to have Petty Kings not Principalities, but then again, Princehood may represent Norman rules accepted by Englishmen? (Furistos of Old Saxons share a similar problem, but here @Urkaun or Uebergold may help - for English I'm at a loss.)

You can maybe raise this on the forum as Uebergold isn't here,

**Comment by Viridianus** [comment]
Here's the thing: since Old Saxons and Saxons seem to use the same titles, this can maybe be postponed till Central Germanic.

**Comment by Charanraja123** [comment]
Considering Old Saxon is practically dead in the game, we can just mod it in and let it be. 
Revisit it if Paradox or any other mod ever use it? 
We don't want to waste time checking something that is not gonna be used in the near future.

**Comment by Viridianus** [comment]
Well, I raised some relevant questions on the forum, but you're free to move your attention elsewhere if desired.

**Comment by Charanraja123** [comment]
Well you have to say whether Brythonic or Frankish.

**Comment by Viridianus** [comment]
Aemr's silent, so Frankish.

**Comment by Charanraja123** [comment]
Question: Is frankish also a dead culture, like old_saxon?

**Comment by Viridianus** [comment]
Yup. And ALSO Germanic. So feel free to skip it till we reach the Germanic cluster.

**Comment by Charanraja123** [comment]
Did we finish Frankish?

**Comment by Viridianus** [comment]
So it would seem... What about Outremer though?

**Comment by Charanraja123** [comment]
Skipped. We barely get through Historical Titles, Alt history is just so hard. Maybe we can revisit it later after we finish stuff

**Comment by Gr770** [comment]
Outremer would use french or church latin.

**Comment by Viridianus** [comment]
Yeah, for them using Medieval Latin actually sounds reasonable. Skipped till Latin. Who's next, Brythonic or Iberian?

**Comment by Charanraja123** [comment]
Brythonic would be my vote

**Comment by Viridianus** [comment]
Cool, then we'll need to speed up Aemr on independent counts :D

**Comment by Viridianus** [comment]
[QUOTE="Aemr, convMessage: 1147957, member: 110511"]Right, so basically I've had a rethink on the Brythonic titles so keep your eye on the sheet today I'm going to do some editing. It should answer your questions.[/QUOTE]

And we have our first "Checked" group as Aemr also confirmed the Gaelic tribal titles!

**Comment by Charanraja123** [comment]
Wow.. Did we actually finish something? xP
